sharding/interface.go

```diff
@@ -74,7 +74,7 @@ type ArgsUpdateNodes struct {
 // NodesShuffler provides shuffling functionality for nodes
 type NodesShuffler interface {
 	UpdateParams(numNodesShard uint32, numNodesMeta uint32, hysteresis float32, adaptivity bool)
-	UpdateNodeLists(args ArgsUpdateNodes) (map[uint32][]Validator, map[uint32][]Validator, []Validator)
+	UpdateNodeLists(args ArgsUpdateNodes) ([][]Validator, [][]Validator, []Validator)
```

Comment:
As seen in other parts returning the same parameter type more than once and in a consecutive manner might lead to mistakenly return wrong variables (of the same type) leading into hard-to-catch bugs. Suggestion (for the next PRs) to return a DTO

Reply:
fixed. Added ResUpdateNodes